CONTRIBUTING.md

```diff
@@ -356,10 +359,27 @@ openssl req\
   -x509\
   -days 365\
   -out chain.crt\
-  -passin env:PASSWORD_ENVIRONMENT_VARIABLE_NAME\
+  -passin env:PLUGIN_SIGN_PRIVATE_KEY_PASS\
   -subj "/C=PL/ST=Krakow/L=Krakow/O=VirtusLab/OU=Git Machete team/CN=www.virtuslab.com/emailAddress=gitmachete@virtuslab.com"
 ```
 Please note that you would have to copy the contents of `private.pem`, `chain.crt`, and `$PASSWORD_ENVIRONMENT_VARIABLE_NAME`, in the [corresponding environment variables](https://app.circleci.com/settings/project/github/VirtusLab/git-machete-intellij-plugin); in order for the Gradle IntelliJ Plugin to pick them up and use them for the plugin signing task, before publishing to the Marketplace.
+For doing so, on a MacOS system you can follow the below instructions:
+
+1. Type the following command for copying the contents of the private key to the clipboard
+```shell
+pbcopy < private.pem
```

Comment:
Just sth like
```shell
export PLUGIN_SIGN_PRIVATE_KEY=$(<private.pem)
export PLUGIN_SIGN_PRIVATE_KEY_PASS=$(<chain.crt)
```

No need to use clipboard

Reply:
Oh, actually this snippet with `export`s can go instead of line 365 above

Reply:
Instead of _Please note that you would have to copy the contents..._

Reply:
Line 365 also includes the link for adding env variables. Do I put it somewhere else?

Reply:
Yeah, sorry for not being clear... I mean, the instructions for **local** tests of signing still aren't complete: a snippet with how to export the env vars and run Gradle would be useful. In fact, I'd  make a clear separation (`####` sections) between:
1. common steps for local (test) signing and setting up CI for (production) signing
2. steps only for local signing
3. steps only for setting up CI for signing